recipes/ww/meta.yaml

```diff
@@ -0,0 +1,29 @@
+{% set name = "ww" %}
+{% set version = "1.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: "{{ version }}"
+
+source:
+  url: https://github.com/ESSS/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: 241fc1a88f30fa6789b56e96d67cf3574da19232e367bf14e44554d2125717be
+
+build:
+  number: 0
+
+test:
+  commands:
+    - ww
+
+about:
+  home: https://github.com/ESSS/ww
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE.txt
+  summary: 'The multiple-workspace batch script'
+
+extra:
+  recipe-maintainers:
+    - igortg
+    - loliveira
```

Comment:
@loliveira do you agree to maintain this recipe?

Reply:
yes

Reply:
No! Pls change loliveira to  lvoliveira =)

Reply:
Ops